week-4/A-objects-intro/exercise-part-1.js

```diff
@@ -6,4 +6,39 @@ can describe with a JavaScript object
 Assign each of them to a separate variable
 
 */
+var cat = {
+    type : "animal",
+    seize : "small",
+    wild : false
+}
+console.log(cat);
 
+var airplane = {
+    type : "object",
+     function: "means of transport",
+    Speed: true

```

Comment:
Just a minor detail: object properties are always camel case which means the first letter is lower case.